rulebook.tex

```diff
@@ -1405,37 +1425,44 @@ \subsubsection{MPS --- During Production Phase}
 In the production phase a production machine performs refinement steps on
 a product such as mounting an additional ring or a cap. Machines
 operate in a transaction style. That is, before using a machine it
-must be prepared for a specific mode. Then the input products can be fed
-into the machine and the refinement step commences. Eventually, after
-the production period is completed, the resulting product is delivered
-on the output lane.
+must be prepared for a specific mode. Most machines take any workpiece
+placed on the input of the machine, perform its operation, and move the
+resulting modified workpiece to the output. Exceptions are \ac{DS}, \ac{BS},
+ and \ac{SS}.
+A \ac{DS} only consumes workpieces, a \ac{BS} can only release (fresh)
+workpieces, whereas a \ac{SS} only does one step (consumption or release)
+at a time.
 
 A light signal mounted near the output lane
 (cf. \refsec{sec:machines}) indicates the state of the machine.
 The light signals are summarized in
 \reftab{tab:machine-lights}.
 
-In the production phase, machines must be prepared to be used by instructing
+Machines must be prepared to be used by instructing
 the refbox. This applies to all machine types.
 If a product from the input is required for the processing step
 (\ac{DS}, \ac{CS}, \ac{RS} and a storage operation at the \ac{SS}),
 then the conveyor belt is started
 after receiving a prepare message. If no workpiece reaches the
 machine operating point (indicated through the middle belt sensor)
-within \SI{45}{\second}, the machine goes to a temporary out-of-order
-state (cf.~\refsec{sec:broken-machine}).
+within \SI{20}{\second}, the machine goes to a temporary out-of-order
+state.
 The \ac{BS} and the retrieval operation at the \ac{SS} cause
 the requested workpiece to be dispensed upon receiving a prepare message.
 After performing an operation at the \ac{BS}, \ac{CS}, \ac{RS} or the \ac{SS}
 (in case of a retrieval) the conveyor belt is started.
 If no workpiece reaches the destination
 point (indicated through the output belt sensor, or in case of the \ac{BS}
-either the input or output belt sensor) within \SI{90}{\second},
+either the input or output belt sensor) within \SI{40}{\second},
 the machine goes in a temporary out-of-order state
 (cf.~\refsec{sec:broken-machine}).
 The workpiece may remain on the machine for an arbitrary time after the
 operation is completed.
 
+If the out-of-order state is reached despite the team operating the machine
```

Comment:
i would restructure this whole topic of machine malfunctioning and recoveries.

I think we should first describe the intended interaction fully, then make a paragraph of how deviations from this are dealt with.
In particular, I would do it like this:
Distinguish between
1) faulty instruction or usage (e.g., instruct without putting something at input) -> should cause broken state
2) technical machine difficulties -> are resolved by referee and field refs in timely manner and will end up with the desired effects. These are no grounds for complaints, as these things can happen equally likely to each team. I would also do not put a fixed time on this (e..g, the previous 20/40 sec deadlines), as it is entirely situation dependent.

Reply:
this also probably should be merged with the broken machine subsection, as it is kinda redundant.

Reply:
makes sense in general to rewrite it like this, however it's a bit difficult given that the operations on different machines vary a lot. I'll try to come up with something

Reply:
i mean yea the happy case varies a bit, but the failure cases are the same across machines

Reply:
not really, but yeah I'll try my best

Reply:
Please check the current version again!

Reply:
reads nicely and coherent now, thanks a lot. Only have a minor comment, see above.